src/main/java/uk/gov/justice/hmpps/prison/repository/sql/ScheduleRepositorySql.kt

```diff
@@ -84,7 +87,7 @@ enum class ScheduleRepositorySql(val sql: String) {
         AND OCA.CRS_SCH_ID = CS.CRS_SCH_ID
                 WHERE CA.AGY_LOC_ID = :agencyId
         AND (OPP.OFFENDER_PROGRAM_STATUS = 'ALLOC'
-                OR (OPP.OFFENDER_PROGRAM_STATUS = 'END' AND OPP.OFFENDER_END_DATE >= :toDate))
+                OR (OPP.OFFENDER_PROGRAM_STATUS = 'END' AND OPP.OFFENDER_END_DATE >= :fromDate))
```

Comment:
have to restrict by the `fromDate` here and then `filter` in the kotlin.  Tried to add in the `SCHEDULE_DATE` into the criteria, but then the explain plan looked really nasty.